src/Angor/Shared/Models/PsbtWrapper.cs

```diff
@@ -0,0 +1,8 @@
+using Blockcore.Consensus.TransactionInfo;
+
+namespace Angor.Shared.Models;
+
+public class PsbtWrapper
```

Comment:
Perhaps PsbtData or PsbtMessage?

Reply:
yeah I also didn't like that name

Reply:
PsbtDto?

Reply:
`PsbtData` or `PsbtInfo`  are my preferences

Reply:
PsbtData it is